### PR TITLE
Fix pending tests.

### PR DESCRIPTION
**Pull request by tcard**

### CODE REVIEW

**Comment by mattheworiordan** [comment]
Nice, looks good :+1:

**Comment by tcard** [comment]
@mattheworiordan @ricardopereira I've added fixes for all pending right now in master. I will add fixes now to each individual pull request that @ricardopereira opened so they can be merged once they're passing.

File: Spec/RealtimeClientPresence.swift [hunk 1]
```diff
@@ -337,37 +366,45 @@ class RealtimeClientPresence: QuickSpec {
...
+                    let channel = client.channels.get("test")
                     channel.onError(AblyTests.newErrorProtocolMessage())
+                    expect(channel.state).to(equal(ARTRealtimeChannelState.Failed))
 
                     waitUntil(timeout: testTimeout) { done in
-                        channel.presence.subscribe { member in
-                            // TODO: missing error
-                            done()
-                        }
+                        channel.presence
```
Comment (ricardopereira) [inline Spec/RealtimeClientPresence.swift]: It doesn't do anything with `presence`.

**Comment by ricardopereira** [comment]
Just a small comment. Looks great!

File: Spec/RealtimeClientPresence.swift (hunk 1 above)
Comment (tcard) [inline Spec/RealtimeClientPresence.swift]: Fixed at 6f98c8f.

**Pull request merged**